tests/Doctrine/Tests/DBAL/Query/Expression/CompositeExpressionTest.php

```diff
@@ -16,30 +16,35 @@ public function testCount() : void
 
         self::assertCount(1, $expr);
 
-        $expr->add('u.group_id = 2');
+        $expr = $expr->with('u.group_id = 2');
 
         self::assertCount(2, $expr);
     }
 
-    public function testAdd() : void
+    public function testAddWith() : void
```

Comment:
Should it be `testWith()` corresponding to the method being tested?

Reply:
Both are tested right now, the legacy `add()` and the new `with()`.

Reply:
Why do we want to have one test for two APIs? It should be structured the way that enables easy removal of the no longer needed test in `master`.

Reply:
Indeed, fixed in 2df2256c731b6cba4d3875c73e04859c670e83be.